hooks/useFeed.ts

```
@@ -0,0 +1,86 @@
+import { useSession } from '@clerk/clerk-expo';
+import { getYearAndWeek } from '../lib/fileNameGenerator';
+import { createClient } from '@supabase/supabase-js';
+
+const supabaseUrl = process.env.EXPO_PUBLIC_SUPABASE_URL!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)